ansible_collections/arista/avd/roles/eos_designs_to_containerlab/tasks/create_avd_node_files.yml

```diff
@@ -0,0 +1,59 @@
+---
+#tasks file for avd nodes
+
+- name: Remove local containerlab output directories
+  file:
+    path: "{{ item }}"
+    state: absent
+  loop:
+    - "{{ containerlab_dir }}"
```

Comment:
since `containerlab_mapping_dir` is configurable, it _could_ be outside of `containerlab_dir`, so you need to delete all the paths

Reply:
Added:

    - "{{ containerlab_config_dir }}"
    - "{{ containerlab_mapping_dir }}"